services/harmony/env-defaults

```diff
@@ -511,3 +511,9 @@ NET2COG_LIMITS_CPU=128m
 NET2COG_LIMITS_MEMORY=512Mi
 NET2COG_INVOCATION_ARGS='./docker-entrypoint.sh'
 NET2COG_QUEUE_URLS='["ghcr.io/podaac/net2cog:SIT,http://sqs.us-west-2.localhost.localstack.cloud:4566/000000000000/net2cog.fifo"]'
+
+OPERA_RTC_S1_BROWSE_IMAGE=ghcr.io/asfhyp3/opera-rtc-s1-browse:test
```

Comment:
We'll want to update the image label here (and below in SERVICE_QUEUE_URLS) before merging to nasa/harmony. I imagine we'll used a fixed version, e.g. `1.0.0`, and use that for both the UAT and Production configs. Then the deployment workflow is ASF releases a new container, PR goes into nasa/harmony to update the version number, change to nasa/harmony is deployed to harmony UAT and validated, then change is deployed to harmony Prod.

Reply:
Per the Harmony team, the image tags here are not the source of truth for what's deployed to each Harmony environment. They're only the image that is deployed the very first time a service is added to an environment. After that teams can manage the specific tag deployed to a specific environment via the Harmony [service-image-tag](https://github.com/nasa/harmony/blob/main/docs/guides/managing-existing-services.md) api